cmd/envtool/envtool.go

```diff
@@ -219,6 +222,118 @@ func setupMongodb(ctx context.Context, logger *zap.SugaredLogger) error {
 	return ctx.Err()
 }
 
+// setupUser creates a user in admin database with supported mechanisms.
+// The user uses username/password credential which is the same as the PostgreSQL
+// credentials.
+//
+// Without this, once the first user is created, the authentication fails
+// as username/password does not exist in admin.system.users collection.
+func setupUser(ctx context.Context, logger *zap.SugaredLogger, postgreSQLPort uint16) error {
```

Comment:
This does not replace [setupUser()](https://github.com/FerretDB/FerretDB/blob/main/integration/setup/setup.go#L334) in integration test.
Because this function creates user in ferretdb database which postgreSQL schema, `postgres://username@127.0.0.1:5432/ferretdb?pool_max_conns=50`.

Each integration test create its own schema like so, `postgres://username@127.0.0.1:5432/testauthentication?pool_max_conns=50`.

Reply:
Thanks for the clarification! I was worried this could jeopardize running tests directly with `go test` for a moment 😄

Reply:
It's still there for SQLite. It only has handler level authentication, and once the first user is created without this code, auth would fail.

It will be handled in a separate PR.